quintype/quintype-node-components
# add separate es modules build for webpack

- upgraded babel to version 7
- added plugin to annotate connect and other HOCs as pure

- upgraded rollup to latest version

- split the build task to
  - clean the dist folder
  - use rollup to build in cjs mode in dist/cjs/
  - use babel to build in es mode in dist/es/

- moved source code to src/ folder to easily run babel

- moved babel config to common location : .babelrc

- fixed typo in ReviewRating component

- contextTypes are needed currently for context to work. Needs a
refactoring to the latest ReactContext API. Post this, the size should
reduce a bit more.
